core/src/ons/util.js

```diff
@@ -228,6 +228,35 @@ util.findFromPath = (path) => {
 };
 
 /**
+ * @param {HTMLElement} container - Page or page-container that implements 'topPage'
+ * @return {HTMLElement|null} - Visible page element or null if not found.
+ */
+util.getTopPage = container => container && (container.tagName.toLowerCase() === 'ons-page' ? container : container.topPage) || null;
```

Comment:
We are already using `topPage` for the Navigator so I think using the same API makes sense. It is not documented for Splitter or Tabbar so we can change it at any time without problems. We could even document it since it could be useful sometimes...